site/styles/_variables.scss

```diff
@@ -46,6 +46,11 @@ $spacers: (
   '5':   ($spacer * 5),    // 50px
 );
 
+// Neutral Colors
+$grey: #999999;
+$grey-000: rgba($grey, 0.24); // Input fields bg
+$grey-100: rgba($grey, 0.34); // Input fields border & error message bg
```

Comment:
What do the `-000` and `-100` represent here?

Reply:
neutral color removed